app/src/main/AndroidManifest.xml

```diff
@@ -9,12 +9,14 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
-        <activity android:name=".SplashScreen">
+        <activity android:name=".SplashScreenActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
                 <category android:name="android.intent.category.LAUNCHER" />
             </intent-filter>
         </activity>
+        <activity android:name=".MusicianAuthenticationActivity" />
```

Comment:
Suggestion: Add label and parent to add the back button menu on the menu bar
<activity
            android:name=".MusicianAuthenticationActivity"
            android:label="Musician Login"
            android:parentActivityName=".SplashScreenActivity">
        </activity>

Reply:
I think we need to consider whether the login pages really need to distinct from one another or not. We just created them mostly so Rae's SplashScreenActivity would not crash. 

The screen after the login screen needs to be distinct based on the type of user we have. But does the login screen itself need to be distinct?